tools/deps_info.py

```diff
@@ -135,6 +137,7 @@
   'emscripten_websocket_set_onmessage_callback_on_thread': ['malloc', 'free'],
   'emscripten_websocket_set_onopen_callback_on_thread': ['malloc'],
   'emscripten_wget_data': ['malloc', 'free'],
+  'emscripten_yield': ['_emscripten_timeout'],
```

Comment:
I think you can/should drop this one.   (`emscripten_yield` is an internal API and it starts with an `_`, but also internal APIs should never appear in this list, since it only contains undefined symbols in user object).

Otherwise, lgtm, although I'm hoping to completely remove this file very soon (!) in with #18905